data-raw/DJFMP.R

```diff
@@ -77,23 +126,26 @@ DJFMP <-  bind_rows(
            is.infinite(Count) ~ "Unknown length",
            is.na(Length)~ "No fish caught",
            TRUE ~ NA_character_), # Add reasoning for an NA lengths (all "No Fish Caught" for FMWT)
-         Count=if_else(is.infinite(Count), Total, Count))%>%
-  filter(Length!=0 | is.na(Length))%>%
+         Count=ifelse(is.infinite(Count), Total, Count))%>%
+  dplyr::filter(Length!=0 | is.na(Length))%>%
   select(-Total, -TotalMeasured, -Group)%>%
-  left_join(DJFMP_stations, by = "Station") %>%
+  left_join(DJFMP_stations, by = c("Station"="StationCode")) %>%
   # Add species names
-  left_join(Species %>%
-              select(USFWS_Code, Taxa) %>%
-              filter(!is.na(USFWS_Code)),
+  left_join(USFWS_Species,
             by=c("OrganismCode"="USFWS_Code")) %>%
   mutate(SampleID=paste(Source, SampleID), # Add variable for unique (across all studies) sampleID
-         Taxa=str_remove(Taxa, " \\((.*)"))%>% # Remove life stage info from Taxa names
+         #Taxa=str_remove(Taxa, " \\((.*)")
```

Comment:
why was this done?

Reply:
Added this back in. Did do a comparison with `all.equal()` and it showed no differences.